### DISCUSSION TITLE
Project

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-guides#383] [General]
I have created a [Guides Project](https://github.com/micronaut-projects/micronaut-guides/projects/1) for managing the guides backlog and in progress. Please add only new guides not improvements to already existing guides. Note: an issue/pr can be attached to multi projects. Thus, you will see I have assigned some guides to the `Micronaut coordination` project and the  Guides project.

@burtbeckwith @ilopmar let me know if you have any concerns.

**Comment by ilopmar** [comment]
:+1:
